tests/ArmTemplates.Tests/Extractor/Abstractions/ExtractorMockerTestsBase.cs

```diff
@@ -74,6 +77,67 @@ public abstract class ExtractorMockerTestsBase : TestsBase
                 ParamBackend = MockParameterizeBackendSettings.ToString(),
                 ExtractGateways = MockExtractGateways.ToString()
             };
-        }  
+        }
+
+        protected ExtractorConsoleAppConfiguration GetDefaultExtractorConsoleAppConfiguration(
+               string sourceApimName = MockSourceApimName,
+               string destinationApimName = MockDestinationApimName,
+               string resourceGroup = MockResourceGroup,
+               string fileFolder = MockFileFolder,
+               string apiName = MockApiName,
+               string multipleAPIs = null,
+               string linkedTemplatesBaseUrl = null,
+               string linkedTemplatesSasToken = null,
+               string linkedTemplatesUrlQueryString = null,
+               string policyXmlBaseUrl = null,
+               string policyXmlSasToken = null,
+               string splitAPIs = null,
+               string apiVersionSetName = null,
+               string includeAllRevisions = null,
+               string baseFileName = null,
+               ServiceUrlProperty[] serviceUrlParameters = null,
+               string paramServiceUrl = null,
+               string paramNamedValue = null,
+               string paramApiLoggerId = null,
+               string paramLogResourceId = null,
+               string serviceBaseUrl = null,
+               string notIncludeNamedValue = null,
+               string paramNamedValuesKeyVaultSecrets = null,
+               int? operationBatchSize = null,
+               string paramBackend = null,
+               string extractGateways = null
+         )
+        {
+
```

Comment:
I prefer using `=> new` ([ExpressionBodyDefined](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/lambda-operator#expression-body-definition)) methods instead of writing usual ones.

not that important - feel free to ignore this, just shared this